src/rp2_common/hardware_clocks/include/hardware/clocks.h

```diff
@@ -354,9 +354,26 @@ void clocks_enable_resus(resus_callback_t resus_callback);
  * \param gpio The GPIO pin to output the clock to. Valid GPIOs are: 21, 23, 24, 25. These GPIOs are connected to the GPOUT0-3 clock generators.
  * \param src  The source clock. See the register field CLOCKS_CLK_GPOUT0_CTRL_AUXSRC for a full list. The list is the same for each GPOUT clock generator.
  * \param div_int  The integer part of the value to divide the source clock by. This is useful to not overwhelm the GPIO pin with a fast clock. this is in range of 1..2^24-1.
- * \param div_frac The fractional part of the value to divide the source clock by. This is in range of 0..255 (/256).
+ * \param div_frac16 The fractional part of the value to divide the source clock by. This is in range of 0..65536 (/65536).
```

Comment:
Presumably the range is `0..65535` ?

Reply:
This still says "This is in range of 0..65536 (/65536)."; I think it needs to say "This is in range of 0..65535 (/65536)." ?